pyworkflow/protocol/launch.py

```diff
@@ -316,22 +229,3 @@ def _run(command, wait, stdin=None, stdout=None, stderr=None):
         p.wait()
 
     return jobId
-
-
-# ******************************************************************
-# *                 Function related to STOP
-# ******************************************************************
-
-
-def _stopLocal(protocol):
```

Comment:
@azazellochg I think this is for stopping protocols when launched to a Queue, not related to MPI

Reply:
_stopLocal() has benn movied inside stop()